content/en/operator-registry/tasks/building-catalog.md

```diff
@@ -4,9 +4,72 @@ linkTitle: "Building a Catalog of Operators"
 date: 2020-03-25
 weight: 3
 description: >
-  Packaging an operator using a Dockerfile 
+  Creating a Catalog image 
 ---
 
+# Manifest format 
+
+We refer to a directory of files with one ClusterServiceVersion as a "bundle". A bundle is typically just a ClusterServiceVersion and the CRDs that define the owned APIs of the CSV, though additional objects may be included.
```

Comment:
I think we should lead with the current [bundle design](https://github.com/operator-framework/operator-registry/blob/master/docs/design/operator-bundle.md) since these docs are for upstream.

Reply:
@njhale my plan was to keep the old bundle format info here, and once we figure out the versioning, and collect all the changes we made (eg catalog polling), we include all of that in the new version of the doc. For the summit itself it feels like it's better to keep things simple, so that we're not hit with the complaints that users need too many things to start out, like we had before.

Reply:
> For the summit itself it feels like it's better to keep things simple, so that we're not hit with the complaints that users need too many things to start out, like we had before.

While I agree with this, I don't see how presenting the current bundle format is at odds with this goal. Maybe I'm missing something, but wouldn't it be better to avoid showing new users a format we don't want them to use from the start?

Reply:
> I think we should lead with the current bundle design since these docs are for upstream.
Rather than leading with the actual proposal, I meant having users generate and working with new-format bundles; e.g. https://github.com/operator-framework/operator-registry#manifest-format

Reply:
@njhale that does make sense, maybe I am missing something. Can we simply replace the old bundle format, with info of the new bundle format, without having to mention opm? 

PS: I am not sure if we have time to include the opm stuff too coz that would require careful review of the newly added info too.

Reply:
I have backed up the opm stuff I had earlier in my [local branch](https://github.com/harishsurf/Operator-Lifecycle-Manager-doc-website/tree/backup-opm). We could reuse this.

Reply:
That would still need to be reviewed.